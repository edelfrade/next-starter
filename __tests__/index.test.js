import React from 'react'
import { render } from '@testing-library/react'

import App from '../pages/index.jsx'

describe('With React Testing Library', () => {
    it('Shows "HOMEPAGE"', () => {
        const { getByText } = render(<App />)

        expect(getByText('HOMEPAGE')).not.toBeNull()
    })
})

describe('With React Testing Library Snapshot', () => {
    it('Should match Snapshot', () => {
        const { asFragment } = render(<App />)

        expect(asFragment()).toMatchSnapshot()
    })
})