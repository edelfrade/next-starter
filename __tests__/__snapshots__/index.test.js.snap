// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With React Testing Library Snapshot Should match Snapshot 1`] = `
<DocumentFragment>
  <div
    class="pages__Home-aoukjq-0 daIxDA"
  >
    <p>
      HOMEPAGE
    </p>
  </div>
</DocumentFragment>
`;
